src/main/java/com/aws/greengrass/easysetup/GreengrassSetup.java

```diff
@@ -286,25 +291,26 @@ public static void main(String[] args) {
         }
     }
 
-    void performSetup() throws IOException, DeviceConfigurationException, URISyntaxException,
-            InvalidEnvironmentStageException {
+    void performSetup()
+            throws IOException, DeviceConfigurationException, URISyntaxException, InvalidEnvironmentStageException {
         // Describe usage of the command
         if (showHelp) {
             outStream.println(SHOW_HELP_RESPONSE);
             return;
         }
         if (showVersion) {
-            // Use getVersionFromBuildMetadataFile so that we don't need to startup the Nucleus which is slow and will
+            // Use getVersionFromBuildMetadataFile so that we don't need to startup the
+            // Nucleus which is slow and will
             // start creating files and directories which may not be desired
-            outStream.println(String.format(SHOW_VERSION_RESPONSE,
-                    DeviceConfiguration.getVersionFromBuildRecipeFile()));
+            outStream
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are using `println()` rather than a dedicated logging facility makes it difficult to monitor the program behavior. We recommend to use a Java logging facility rather than `System.out` or `System.err`.

[Learn more](https://cwe.mitre.org/data/definitions/398.html)